### ISSUE TITLE
Narrator does not announce the focused ListViewItem when the user returns focus to the ListView after deleting the ListViewItem.

### ISSUE THREAD

**Issue by SergeySmirnov-Akvelon** [dotnet/winforms#6238]
.NET Core Version: 
- .NET Core SDK: 7.0.0-alpha.1.21574.1

Have you experienced this same bug with .NET Framework?: 
- No (the issue is reproduced since .NET Core 5.0)

**Repro steps:**
1. Create a `Form` with a `ListView` and a couple of `Buttons`.
2. Add a couple of `Columns` to the `ListView` and set the view mode to `Details`.
3. Make the first `Button` a button for adding `ListViewItems` to the `ListView`.
4. Make the second `Button` a button to delete `ListViewItems` from the `ListView`.
5. Launch the application.
6. Add a couple of `ListViewItems` using the first `Button`.
7. Set focus to the first `ListViewItems` and press the `Button` to delete.
8. Use `Tab` to return focus to the `ListView`.

**Actual behavior:**
The Narrator announces nothing.
![6238](https://user-images.githubusercontent.com/23376742/143583678-e51f1846-977f-41bd-9bdb-92f005215135.gif)

**Expected:**
The Narrator should announce data about the ListView.
![image](https://user-images.githubusercontent.com/23376742/143584834-b96b3b16-ef83-481b-84bf-bd382d2fecb3.png)

**Comment by NikitaSemenovAkvelon** [comment]
In progress

**Comment by NikitaSemenovAkvelon** [comment]
I found a bad commit that broke the behavior of announcing in NET 5. But currently, behavior looks the same as in NET5 before the bad commit. 
![wxxDvfVUVt](https://user-images.githubusercontent.com/109065597/181711932-b51e7110-b2f7-4fa5-abaa-d739d07842c1.gif)

**Comment by NikitaSemenovAkvelon** [comment]
@Olina-Zhang could you test it, please?

**Comment by Olina-Zhang** [comment]
@NikitaSemenovAkvelon tested this issue in the latest .Net 7.0 build: 7.0.100-rc.1.22402.10, it doesn't reproduce now: after deleting listview item, then enter listView again, Narrator can announce it.
![ListView](https://user-images.githubusercontent.com/26474449/182753635-185e3f95-ec4b-4847-834f-5438e96b49ee.gif)

**Comment by NikitaSemenovAkvelon** [comment]
@merriemcgaw may we close this issue as not reproduced?

**Comment by merriemcgaw** [comment]
Yes, let's close this.
